man/tallow.patterns.5.md

```diff
@@ -0,0 +1,91 @@
+## tallow.patterns
+
+Tallow pattern matching configuration files.
+
+
+## SYNOPSIS
+
+tallow(1) configures patterns to match journal entries and banning thresholds
```

Comment:
I'd say something like: 

`tallow` uses regular expressions to match journal events and extract the IP address in them. The pattern matching configuration files can be used to extend the patterns that tallow can recognize. Tallow reads these files when it starts up and includes all the rules found in them.